src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -411,6 +414,16 @@ public ProjectState UpdateOutputPath(string outputFilePath)
             return this.With(projectInfo: this.ProjectInfo.WithOutputFilePath(outputFilePath).WithVersion(this.Version.GetNewerVersion()));
         }
 
+        public ProjectState UpdateOutputRefPath(string outputRefFilePath)
```

Comment:
"UpdateOutput**Ref**FilePath" to be consistent with everything else?

Reply:
Thanks! Missed that when I re-did this change on dev15.7.x.

Reply:
Actually, this is consistent with `UpdateOutputPath` right above this method.

Reply:
FWIW, I'll go ahead and change this particular method.